net/pfSense-pkg-freeradius2/files/usr/local/pkg/freeradius.inc

```diff
@@ -4517,6 +4531,15 @@ function freeradius_validate_eap($post, &$input_errors) {
 		$input_errors[] = "The 'Maximum Sessions Tracking Per Server' field must contain an integer value.";
 	}
 
+	// Certificates for TLS
+	if ($post['ssl_ca_cert'] == 'none') {
```

Comment:
I've been thinking about grabbing the code from the stunnel package to generate a self-signed certificate that shows in the pfSense cert manager on install (https://github.com/pfsense/FreeBSD-ports/blob/devel/security/pfSense-pkg-stunnel/files/usr/local/pkg/stunnel.inc#L90), but yeah, I'd rather get out something that at least points users to the required configuration now and polish things later.

Definitely a good idea to move the settings from the EAP tab, kinda weird place for that.